tests/integration/components/frost-textarea-test.js

```diff
@@ -293,6 +293,20 @@ describe(test.label, function () {
       this.$('textarea').val(),
       'input value cleared'
     ).to.eql('')
+
+    run.later(() => {
```

Comment:
There actually do not appear to be any other test cases in this test suite that verify that the the `is-clear-visible` and `is-clear-enabled` classes are every applied correctly in the state in which clearing the input value is possible.  Therefore these tests would still pass even if the mechanism to apply the classes being tested for was broken.  I'm not saying that this test case is where this should be asserted but please add an additional test case to this test suite asserting the opposite positive assertions of the ones you have added here.

Reply:
I completely missed that such tests already exist at https://github.com/ciena-frost/ember-frost-core/blob/master/tests/integration/components/frost-text-test.js#L253-L263